### PR TITLE
Proper trademarking of IBM and third-party names

### LINKED ISSUE #726: Add trademarks
The first instance of every trademark should be marked on every page.  I don't think there's an automated way to do that with MD, so we need to go through and mark the first instance of every trademarked term manually. 

There's a lot of good information here (including a link to the list of trademarked terms): https://www.ibm.com/docs/en/technical-content?topic=standards-trademarks 

To code a trademark attribution in Markdown, add the following after the word that needs the attribution:

Registered trademarks, use `&reg;` to get &reg;
Nonregistered trademarks, use `&trade;` to get &trade;
Use the trademark symbol on the first occurrence only in each topic in which the trademark appears.

Do not include trademarks in headings. The value will be displayed as a &trade; or &reg; rather than being interpreted into a symbol.

### PR DESCRIPTION
**Pull request by abbycross**
Closes #726 

The first instance on a page of every trademarked name (whether IBM or third-party) should be marked with the trademark symbol (except in headings, where the code will not render as a symbol). 

Outstanding questions: Where does the trademark symbol occur if the following are the first instance on a page?:
(editing to add: preliminary guidance directs us as follows: )

- [ ] IBM Quantum&trade; systems
- [ ] Qiskit&reg; Runtime service
- [ ] Qiskit&reg; Runtime
- [ ] Qiskit IBM Provider (still unknown)
- [ ] Qiskit&reg; Patterns
- [ ] Qiskit&reg; Experiments
- [ ] Qiskit&reg; 1.0
- [ ] Qiskit&reg; Runtime IBM Client
- [ ] Qiskit&reg; Runtime IBM REST API
- [ ] IBM Quantum&trade; Network

References: 
https://www.ibm.com/brand/experience-guides/quantum/voice#usage
https://www.ibm.com/legal/copyright-trademark

### Progress

- [x] Migration guides
- [x] Start
- [x] Build
- [x] Transpile
- [x] Verify
- [x] Run
- [x] Support

### CODE REVIEW

**Comment by abbycross** [comment]
Still waiting on IP Legal to confirm certain naming conventions but would like to merge this for now, to have a good-faith first pass at trademarks.

File: docs/run/fair-share-queue.mdx [hunk 1]
```diff
@@ -10,7 +10,7 @@ When you submit a job to a quantum system, it enters the scheduler for the speci
 
 ## Fair-share terms
 
-*   **Provider:** An entity providing access to quantum computing. IBM Quantum Platform and IBM Cloud® are providers of Qiskit Runtime services.
+*   **Provider:** An entity providing access to quantum computing. IBM Quantum&reg; Platform and IBM Cloud® are providers of Qiskit&reg; Runtime services.
```
Comment (Eric-Arellano) [inline docs/run/fair-share-queue.mdx:13]:
```suggestion
*   **Provider:** An entity providing access to quantum computing. IBM Quantum&reg; Platform and IBM Cloud&reg; are providers of Qiskit&reg; Runtime services.
```

File: docs/transpile/set-optimization.mdx [hunk 2]
```diff
@@ -149,15 +149,6 @@ Adjusting the optimization level can change other aspects of the circuit too, no
 
 ## Next steps
 
-<Admonition type="tip" title="Recommendation">
```
Comment (Eric-Arellano) [inline docs/transpile/set-optimization.mdx:152]: Oopsies with us repeating this section.

**Review by Eric-Arellano — APPROVED** [review]
Amazing attention to detail. Thank you!

**Comment by abbycross** [comment]
Attribution for IBM: 

> IBM and the IBM logo are trademarks of International Business Machines Corporation, registered in many jurisdictions worldwide. Other product and service names might be trademarks of IBM or other companies. A current list of IBM trademarks is available on ibm.com/trademark.

**Review by beckykd — APPROVED** [review]
Thanks Abby!  This was a ton of work!  I loved all the typo fixes as well!

**Pull request merged**